File: ResearchCruiseApp-API/ResearchCruiseApp-API/Application/DTOs/DataTypes/Publication.cs
namespace ResearchCruiseApp_API.Application.DTOs.DataTypes;


public class Publication
{
    public string Category { get; set; } = null!;

    public string DOI { get; set; } = null!;

    public string Authors { get; set; } = null!;

    public string Title { get; set; } = null!;

    public string Magazine { get; set; } = null!;
    
    public int Year { get; set; }
    
    public int MinisterialPoints { get; set; }
}

File: frontend/src/features/core/models/Publication.ts
export type Publication = {
  id: string;
  category: string;
  doi: string;
  authors: string;
  title: string;
  magazine: string;
  year: string;
  ministerialPoints: string;
};

export type UserPublication = {
  id: string;
  userId: string;
  publication: Publication;
};
